pmm/aws-staging-start.groovy

```diff
@@ -288,8 +288,10 @@ pipeline {
 
                             if [[ \$CLIENT_VERSION = dev-latest ]]; then
                                 sudo yum -y install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
-                                sudo percona-release enable original testing 
+                                sudo percona-release enable original testing
```

Comment:
I would suggest adding  `sudo percona-release disable all` as by default we enable our main repo

Reply:
how about I use `enable-only`, `sudo percona-release enable-only original testing`

Reply:
my suggestions there  use  "as much as possible  real user conditions" 
To get all problems user can have. 
I haven't  seen we  recommend in our Documentation  `disable all`   much